cmd/worker/clickhouse/clickhouse.go

```diff
@@ -74,19 +74,24 @@ func New(ctx context.Context, host string) (*Client, error) {
 //go:embed migrations/*.sql
 var migrations embed.FS
 
-func (c *Client) ApplyMigrations(ctx context.Context) (int, error) {
+func (c *Client) ApplyMigrations(ctx context.Context) (int, int, error) {
```

Comment:
Nit: Maybe this is just me but it's unclear what the return values represent if there's just two integers side by side. Named return values would help here